src/ResourceManager/Sql/Commands.Sql/Database Backup/Cmdlet/RemoveAzureRmSqlDatabaseLongTermRetentionBackup.cs

```diff
@@ -0,0 +1,208 @@
+            Position = 0,
+            ValueFromPipeline = true,
+            HelpMessage = "The Resource ID of the Database Long Term Retention Backup to remove.")]
+        [ValidateNotNullOrEmpty]
+        public string ResourceId { get; set; }
+
+        /// <summary>
+        /// Defines whether it is ok to skip the requesting of rule removal confirmation
+        /// </summary>
+        [Parameter(HelpMessage = "Skip confirmation message for performing the action")]
+        public SwitchParameter Force { get; set; }
+
+        /// <summary>
+        /// Get the entities from the service
+        /// </summary>
+        /// <returns>The list of entities</returns>
+        protected override IEnumerable<AzureSqlDatabaseLongTermRetentionBackupModel> GetEntity()
+        {
+            return ModelAdapter.GetDatabaseLongTermRetentionBackups(
+                LocationName,
+                ServerName,
+                DatabaseName,
+                BackupName,
+                null,
+                null);
+        }
+
+        /// <summary>
+        /// No user input to apply to model
+        /// </summary>
+        /// <param name="model">Model retrieved from service</param>
+        /// <returns>The model that was passed in</returns>
+        protected override IEnumerable<AzureSqlDatabaseLongTermRetentionBackupModel> ApplyUserInputToModel(
+            IEnumerable<AzureSqlDatabaseLongTermRetentionBackupModel> model)
+        {
+            return model;
+        }
+
+        /// <summary>
+        /// No changes to persist to server
+        /// </summary>
+        /// <param name="entity">The output of apply user input to model</param>
+        /// <returns>The input entity</returns>
+        protected override IEnumerable<AzureSqlDatabaseLongTermRetentionBackupModel> PersistChanges(
+            IEnumerable<AzureSqlDatabaseLongTermRetentionBackupModel> entity)
+        {
+            ModelAdapter.RemoveDatabaseLongTermRetentionBackup(LocationName, ServerName, DatabaseName, BackupName);
+            return entity;
+        }
+
+        /// <summary>
+        /// Entry point for the cmdlet
+        /// </summary>
+        public override void ExecuteCmdlet()
+        {
+            if (InputObject != null)
+            {
+                LocationName = InputObject.LocationName;
+                ServerName = InputObject.ServerName;
+                DatabaseName = InputObject.DatabaseName;
+                BackupName = InputObject.BackupName;
+            }
+            else if (!string.IsNullOrWhiteSpace(ResourceId))
+            {
+                string[] tokens = ResourceId.Split(new[] { '/' }, StringSplitOptions.RemoveEmptyEntries);
+
+                LocationName = tokens[LocationNameIndex];
+                ServerName = tokens[ServerNameIndex];
+                DatabaseName = tokens[DatabaseNameIndex];
+                BackupName = tokens[BackupNameIndex];
+            }
+
+            if (!Force.IsPresent && !ShouldProcess(
```

Comment:
Use this syntax: https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Profile/Commands.Profile/Default/ClearAzureRmDefault.cs#L52

Reply:
Done. What does ShouldProcess do?

Reply:
Should process allows the user to specify the "Confirm" parameter, which will prompt them as soon as it hits this shouldprocess block.